test/e2e/specs/specs-playwright/wp-widgets__spec.ts

```diff
@@ -0,0 +1,50 @@
+import { DataHelper, LoginFlow, SidebarComponent, setupHooks } from '@automattic/calypso-e2e';
+import { Page } from 'playwright';
+
+const gutenbergUser =
```

Comment:
I think we might not need that at all in this test, as this is not really GB specific - and what caused the bug wasn't really related to GB, though :thinking_face: so I might as well just remove it.

Reply:
I don't understand fully what and how the Gutenberg tests in TC are set up, so I might miss my mark here. Feel free to ignore if I don't make any sense 😆 

I think it would be best to move away from the practice that's seen often in the Selenium suite, to use a different user depending on the environment. I think it might make more sense to [parametrize the test](https://github.com/Automattic/wp-calypso/blob/trunk/test/e2e/specs/specs-playwright/wp-media__upload-spec.js#L24-L27) and test it under both `gutenbergSimpeSiteEdgeUser` and `gutenbergSimpleSiteUser`.

This way we have a proper baseline that isn't dependent on an environment variable being set.

Reply:
I like that pattern! I'll keep that in mind for any eventual playwright GB tests I might write in the future. For now, this ended up not being needed at all, as this test is not really GB-specific :)